typewise-alert.c

```diff
@@ -1,5 +1,7 @@
 #include "typewise-alert.h"
 #include <stdio.h>
+LimitDetail_st classifyTemperatureBreach[] = {{LOWER_LIMIT, PASSIVE_COOLING_UPPER_LIMIT},{LOWER_LIMIT,HI_ACTIVE_COOLING_UPPER_LIMIT},{LOWER_LIMIT,MED_ACTIVE_COOLING_UPPER_LIMIT}};
+AlertRetStatus(*AlertType[])(BreachType) = {sendToController, sendToEmail, sendToConsole};
```

Comment:
Your tests will pass even when sendToController and sendToEmail are interchanged. Strengthen the tests as discussed today.